Patch main

c4rt0 opened the pull request:
update main.tf

jbtrystram committed:
coreos: update to f40

jbtrystram committed:
increase users to 24 and add launch script

c4rt0 committed:
Update main.tf with valid credentials

Co-authored-by: jbtrystram@redhat.com

c4rt0 commented:
Ok, I think I messed up something here with the last push. 
cc. @jbtrystram

jbtrystram commented on fedora-coreos/provisioning/main.tf:
```diff
@@ -11,7 +11,9 @@ terraform {
 }
 
-provider "aws" {}
+provider "aws" {
+  region = "eu-central-1"
+}
```
Remove this change, it's already configurable through https://github.com/coreos/coreos-tutorials/commit/e8945b44a80d39910aa4ead9c37424b662908b8c

c4rt0 replied:
Implemented in https://github.com/coreos/coreos-tutorials/commit/d9ea4e5f3d1cd18f80e1c3334ae7aa4f492af392.

jbtrystram commented on fedora-coreos/provisioning/main.tf:
```diff
@@ -34,15 +36,15 @@ resource "aws_instance" "fcos-lab-instance" {
     Name = "fcos-lab"
   }
-  ami           = "ami-0ea3c2efdcead938c"
+  ami           = "ami-06128ecf4b4101217"
```
maybe port [this feature](https://github.com/coreos/coreos-tutorials/commit/97376716dce47080dd9e8f016753c55c5c79da96) here ?

c4rt0 replied:
Completed with https://github.com/coreos/coreos-tutorials/commit/d9ea4e5f3d1cd18f80e1c3334ae7aa4f492af392.

jbtrystram commented:
>Ok, I think I messed up something here with the last push.

@c4rt0 I still have my stuff locally, that's fine. 
I fyou want to add new changes makes sure doing `git pull -r` first :)

travier commented on fedora-coreos/provisioning/fcos-lab-tutorial.bu:
```diff
@@ -62,24 +72,28 @@ storage:
+    - path: /usr/local/bin/launch_coreos_vm
+      mode: 0755
+      contents:
+        source: https://raw.githubusercontent.com/jbtrystram/devconf24-coreos-workshop/main/launch_coreos_vm
```
Let's push this to this repo?

jbtrystram replied:
Done. It was overwritten by the force push

jbtrystram committed:
provisionning: move launch script to this repo

jbtrystram committed:
provisionning: add a dummy ignition file

This is a required workaround for the `lauch_coreos_vm` script
otherwhise it runs into expanding variables issue when no ignition
config is provided so we pass this empty one

c4rt0 committed:
Port automatic ami recognition

ref: https://github.com/coreos/coreos-tutorials/commit/97376716dce47080dd9e8f016753c55c5c79da96

jbtrystram committed:
DNM: propper script location until merged

jbtrystram committed:
launch_coreos_vm script remove set -x

jbtrystram committed:
add vim

travier commented on fedora-coreos/provisioning/fcos-lab-tutorial.bu:
```diff
@@ -58,28 +68,36 @@ storage:
+    - path: /usr/local/bin/launch_coreos_vm
+      mode: 0755
+      contents:
+        source: https://raw.githubusercontent.com/jbtrystram/coreos-tutorials/Patch_main/fedora-coreos/provisioning/launch_coreos_vm
```
```suggestion
        source: https://raw.githubusercontent.com/coreos/coreos-tutorials/Patch_main/fedora-coreos/provisioning/launch_coreos_vm
```

travier reviewed (approved):
Let's get this merged now that the workshop is done?

c4rt0 committed:
Update fedora-coreos/provisioning/fcos-lab-tutorial.bu

Co-authored-by: Timothée Ravier <tim@siosm.fr>

c4rt0 commented:
I just found this months later - since I'm not an owner <b> ping'</b>ing you folks to merge this @jbtrystram, @dustymabe

jbtrystram commented:
i can't either. 
There is one commit that needs to be dropped before merging

c4rt0 commented:
Since I (thankfully 😆) have no access to your repo, let's close this one on behalf of https://github.com/coreos/coreos-tutorials/pull/3
